### PR TITLE
add property schema for kubectlmage

### PR DESCRIPTION
**Pull request by vxav**
This PR:

- adds/changes/removes etc

### Testing

Description on how {APP-NAME} can be tested.

- [ ] fresh install works
- [ ] upgrade from previous version works

#### Other testing

Description of features to additionally test for {APP-NAME} installations.

- [ ] check reconciliation of existing resources after upgrading
- [ ] X still works after upgrade
- [ ] Y is installed correctly

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.
- [ ] Update Lastpass values if required.

### CODE REVIEW

File: helm/cluster-cloud-director/values.schema.json
Comment (marians) [inline helm/cluster-cloud-director/values.schema.json:187]: Let's please use sentence case

```suggestion
            "title": "Kubectl image",
```

File: helm/cluster-cloud-director/values.schema.json
Comment (marians) [inline helm/cluster-cloud-director/values.schema.json:188]: Thanks for providing a description! Can we explain _what_ it's being used for?

File: helm/cluster-cloud-director/values.schema.json
Comment (marians) [inline helm/cluster-cloud-director/values.schema.json:189]: Pretty sure this should be `properties`, as you are naming the properties specifically.

```suggestion
            "properties": {
```

Comment (vxav) [reply helm/cluster-cloud-director/values.schema.json:189]: yes

File: helm/cluster-cloud-director/values.schema.json
Comment (vxav) [reply helm/cluster-cloud-director/values.schema.json]: Would [this](https://github.com/giantswarm/cluster-cloud-director/pull/68/commits/f1fee8e223eb9de822dad9af7721a70f0435e03c) be explicit enough?
It is used by [coredns](https://github.com/giantswarm/cluster-shared/blob/9b3382b2737900505f7eee64677db61be7f0a70d/helm/cluster-shared/templates/_coredns.tpl#L279) for now but I suspect it could be used by other components in the future.

File: CHANGELOG.md
Comment (marians) [inline CHANGELOG.md:10]: The word "items" suggests that this is an array property.

```suggestion
- Add schema for `.kubectlImage`.
```

Comment (vxav) [reply CHANGELOG.md:10]: I just reused that of your previous change 😅

**Review by marians — APPROVED** [review]

**Comment by vxav** [comment]
Waiting for a @giantswarm/team-rocket review before merging.

File: helm/cluster-cloud-director/values.schema.json
Comment (marians) [reply helm/cluster-cloud-director/values.schema.json]: Totally

**Comment by vxav** [comment]
Yolo

File: helm/cluster-cloud-director/values.yaml
Comment (kopiczko) [inline helm/cluster-cloud-director/values.yaml:103]: I think we should add a comment that it's used in `cluster-shared`. Otherwise it's very cryptic.

**Pull request merged**
